main.py

```
@@ -445,56 +577,491 @@ def get_config(user: str = Depends(authenticate)):
+    except ImportError:
+        logger.warning("wmi no disponible, información de hardware limitada")
+        return HardwareInfo(
+            cpu_name="Desconocido",
+            cpu_cores=0,
+            cpu_threads=0,
+            total_ram_gb=0,
+            motherboard="Desconocido",
+            os_version=platform.platform(),
+            gpus=[]
+        )
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)